### PR TITLE
[Resources] Fix tenant leve What-If examples

### PR DESCRIPTION
**Pull request by shenglol**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/examples/PostDeploymentWhatIfOnTenant.json [hunk 1]
```diff
@@ -18,59 +18,13 @@
...
-                "type": "Microsoft.Authorization/policyAssignments",
-                "name": "myPolicyAssignment2",
-                "location": "westus2",
-                "properties": {
-                  "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/myManagementGroup/providers/Microsoft.Authorization/policyAssignments/myPolicyDefinition",
-                  "scope": "/subscriptions/00000000-0000-0000-0000-000000000002",
-                  "enforcementMode": "Default"
-                }
+                "id": "/providers/Microsoft.Management/managementGroups/myManagementGroup",
+                "apiVersion": "2019-11-01",
```
Comment (raych1) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/examples/PostDeploymentWhatIfOnTenant.json:25]: Should this version align to rest api version?

File: specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/PostDeploymentWhatIfOnTenant.json [hunk 2]
```diff
@@ -18,59 +18,13 @@
...
-                "type": "Microsoft.Authorization/policyAssignments",
-                "name": "myPolicyAssignment2",
-                "location": "westus2",
-                "properties": {
-                  "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/myManagementGroup/providers/Microsoft.Authorization/policyAssignments/myPolicyDefinition",
-                  "scope": "/subscriptions/00000000-0000-0000-0000-000000000002",
-                  "enforcementMode": "Default"
-                }
+                "id": "/providers/Microsoft.Management/managementGroups/myManagementGroup",
+                "apiVersion": "2019-11-01",
```
Comment (raych1) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/PostDeploymentWhatIfOnTenant.json:25]: Same here, should this version align to rest api version?

File: specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/examples/PostDeploymentWhatIfOnTenant.json (hunk 1 above)
Comment (shenglol) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/examples/PostDeploymentWhatIfOnTenant.json:25]: No, this is the version of the resource type `Microsoft.Management/managementGroups`.

File: specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/PostDeploymentWhatIfOnTenant.json (hunk 2 above)
Comment (shenglol) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/PostDeploymentWhatIfOnTenant.json:25]: Answered above.

**Review by raych1 — APPROVED** [review]

**Pull request merged**